docs/models.rst

```diff
@@ -0,0 +1,8 @@
+.. _new_page:
+
+Models
+==============
+
+.. doxygenenum:: BSMPT::ModelID::ModelIDs 
```

Comment:
Do we want the enums here? Or just a list of the implemented models and a link to their extra page? In doxygen we have the LaTeX code for the potential, maybe show that there?

Reply:
Not really, I was just playing around to see what sphinx could do. I will improve this